microshift_updating/microshift-update-options.adoc

```diff
@@ -48,10 +55,19 @@ To understand more about Greenboot, see the following documentation:
 
 [id="microshift-update-options-manual-rpm-updates_{context}"]
 === Manual RPM updates
-You can update {microshift-short} manually on a non-OSTree system such as {op-system-base-full} by downloading and updating the RPMs. To complete this update type, use the subscription manager to access the repository containing the new RPMs. To begin a manual RPM update, use the procedures in the following documentation:
+You can update {microshift-short} manually on a non-OSTree system such as {op-system-base-full} by updating the RPMs.
+
+* To complete this update type, use the subscription manager to access the repository containing the new RPMs.
```

Comment:
This is a minor nit, but you are using subscription manager to enable the repository, not really accessing the repository.